### PR TITLE
fix(spaces): Display using infinite scroll

### LINKED ISSUE #290: infinite scroll is not working on /<username>/_spaces

### PR DESCRIPTION
**Pull request by VineetReynolds**
This ensures the spaces listing is not limited to
only first 20 results, or something thereof. Further spaces are
displayed on scroll. Fixes #290

### CODE REVIEW

**Comment by VineetReynolds** [comment]
Please don't merge, as this is dependent on https://github.com/fabric8-ui/ngx-fabric8-wit/pull/52

File: src/app/profile/spaces/spaces.component.html [hunk 1]
```diff
@@ -29,18 +31,22 @@
...
-              <h2><a [routerLink]="['/', s.relationalData.creator.attributes.username, s.attributes.name]">{{s.attributes.name}}</a></h2>
-              <p>

{{s.description}}</p>
-            </td>
-          </tr>
-        </table>
+      <div class="contents posRel height0">
+        <div class="spaces-list-page flex-container in-column-direction flex-grow-1">
+          <div class="spaces" class="spaces-list-wrap" almInfiniteScroll (eachElementHeightInPx)='contentItemHeight' (initItems)='initSpaces($event)' (fetchMore)='fetchMoreSpaces($event)'>
```
Comment (joshuawilson) [inline src/app/profile/spaces/spaces.component.html]: Why do you have "class" in here twice?

Comment (VineetReynolds) [inline src/app/profile/spaces/spaces.component.html]: The first is not required. Will fix.

Comment (joshuawilson) [inline src/app/profile/spaces/spaces.component.html]: you can have both classes listed just not the attribute "class"

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**